types/aws-lambda/index.d.ts

```diff
@@ -1,4 +1,4 @@
-// Type definitions for AWS Lambda 8.10
+// Type definitions for AWS Lambda 9.0
```

Comment:
No major (or minor) version bump please. I don't think this PR has a breaking change, but even if it did, it wouldn't be worth a version bump: either people take the breaking change and fix it if it's a simple fix and something that was glaringly wrong with their code, or we don't make the change. So far, only AWS runtime updates that have been potentially breaking changes to the handler API have qualified, hence the version being the last runtime version to do so.

I've had a vague plan to split out the runtime and various services to make it less onerous to make breaking changes to an API, but there's a whole bunch of possible issues there (esp. with what the existing package will do) and it's a bunch of work for dubious gain for the end user.